### PR TITLE
template: Fix `GetVBR` for systems with MMU

### PR DESCRIPTION
**Pull request by yupferris**
Previously, the supervisor portion of `GetVBR` would be placed in the data hunk, which goes bang on systems with an MMU. So, let's make sure that code is placed in the code hunk instead so it will be properly executable.

### CODE REVIEW

**Comment by yupferris** [comment]
Fixes #256.

**Comment by yupferris** [comment]
… and actually reading #256 it’s entirely possible I’ve misdiagnosed the problem 😅 either way, this should be a proper fix (and it makes the template work on my tf1260 😊).

**Comment by yupferris** [comment]
I did some more digging, and indeed I've misdiagnosed the issue:
 - Looking through the  M68060 User Manual, there's clearly only supervisor-only and write protections provided by the MMU, no execute protections.
 - Looking at the disassembly of the original code, the code is indeed written to memory to initialize `getvbr` right before it's used: bilde so it's extremely likely the code is sitting in a dirty line in the data cache this point, and not committed to memory (much less in the icache, for that matter). And, as suggested in #256, clearing caches (via `CacheClearU`) right after the copy indeed fixes the issue on my setup as well.

tl;dr, #256 has a correct description of the issue, so I'll update my PR title/description and commit message. However, my solution should still be preferred, as it's cleaner (keeps code in a function, not as raw data words), avoids an unnecessary copy (to initialize the local variable), and avoids an unnecessary cache clear (expensive).

**Comment by yupferris** [comment]
and, lol, long time using github or something, I've made a PR only in my own repo! OK. Nice job Jake. I guess I can keep this shame to myself then :)

**Comment by yupferris** [comment]
Proper commit/PR here: https://github.com/BartmanAbyss/vscode-amiga-debug/pull/267
